src/app/core/user/user-security/user-security.component.html

```diff
@@ -0,0 +1,43 @@
+<form [formGroup]="form">
+  <mat-form-field>
+    <mat-label i18n="label of username input">Username</mat-label>
+    <input matInput type="text" formControlName="username">
+  </mat-form-field>
+  <mat-form-field>
```

Comment:
UX-wise this might be confusing (why are there two emails for a single user?) I suppose it's not that big of an issue since (correct me if I'm wrong) this is only seen by admins who should know what they are doing.

Reply:
Can you explain what you mean with the two emails? There should only be one email field with a label connected to it.

Reply:
Ah no, these are actually the same email. In the `UserAccountComponent` you can see the email for the currently logged in user (you) which everybody can see and change for themselves. The `UserSecurityComponent` (I am happy for any better name suggestion) shows the same email, but for any user that is registered to the system (only seen by managers with the required permissions).
So the email and the meaning is exactly the same - the email with which a user is registered and can reset the password.

Reply:
Ah, I understand now. In this case, the two fields should, however, always show the same information. Currently, at least in the demo app, the email under `Security` is empty (while the username isn't) which has given me the impression that the two emails are different to begin with.

Reply:
Ah now I get it, yes the demo data has to be adjusted 😄 good point, thanks!